layouts/partials/header/basic.html

```diff
@@ -0,0 +1,178 @@
+<div style="padding-left:0;padding-right:0;padding-top:2px;padding-bottom:3px"
```

Comment:
What is customised here?

Reply:
I wanted to add EVERSE .svg figure for "Home page button"

<img width="123" alt="image" src="https://github.com/graeme-a-stewart/EVERSE-ResearchSoftware.github.io/assets/44325851/ba7aaacc-fec3-44b8-8515-bf12d064f660">